scripts/build_and_push_images.sh

```diff
@@ -20,7 +20,7 @@ IMG="$REPO/$OP_NAME":$VERSION
 IMAGE_TAG_BASE="$REPO/$OP_NAME"
 
 
-AGENT_IMAGE="$OP_NAME-agent"
+AGENT_IMAGE="osp-director-agent"
```

Comment:
This is the format used everywhere else as opposed to `osp-director-operator-agent`